arcade/experimental/__init__.py

```diff
@@ -1,8 +1,8 @@
 """
 Experimental stuff. API may change.
 """
-from .video_player import VideoPlayer
-from .video_player import VideoPlayerView
+from .video_player import VideoPlayer, VideoPlayerView
+from .video_cv2 import VideoPlayerCV2, CV2PlayerView
```

Comment:
Wrap the cv2 imports in a try/except ImportError. The module should only be available if you have cv2 installed.

Reply:
I'm not sure if what I've done is kosher. I was struggling with not putting out an error message for everyone every time they import arcade if they don't have cv2, but also having an error message when the user tries to use the class without the right dependencies. It does mean that what should be the CV2PlayerView class becomes a function callback, which feels like a hack.. but it works.

... And the tests have failed, so I guess it's too hacky. Advice? Just do `except: pass` and have it fail silently?

Reply:
Just `pass` for now is fine. It's still something in experimental. It can be improved of we move it down into main namespace.

Reply:
Fixed in [8df8bd6](https://github.com/pythonarcade/arcade/pull/1786/commits/8df8bd65491a163fde791da3e4d4006c8e9c0d55)